src/public/less/pages/migrating.less

```diff
@@ -46,14 +46,18 @@
 		display: flex;
```

Comment:
Somehow I get like 1 FPS on the animation here, any ideas? It looks terrible.

![2022-08-22_14-25-18](https://user-images.githubusercontent.com/6192491/185920946-57995311-71b5-4ff4-a15f-9be199c1e30c.gif)

Reply:
I left the animation the same as it was before my changes. I thought it was checked, i can fix it, how should it look like?